.changeset/quick-toys-see.md

```
@@ -0,0 +1,14 @@
+---
+"@belgattitude/eslint-config-bases": patch
+---
+
+Re-release
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)